docs/compatibility_matrix.md

```diff
@@ -0,0 +1,14 @@
+# Compatibility Matrix
+
+This document shows the compatibility of Scalar DB with supported databases.
+
+## Scalar DB Compatibility
+
+| Scalar DB Version   | Cassandra 3.11.10 | Cassandra 3.0.24    | Cassandra 2.2.19   | Cosmos DB | DynamoDB  | MySQL 8.0.25  | MySQL 2.7.34  | PostgreSQL 13.3    | PostgreSQL 12.7   | PostgreSQL 11.12  |
```

Comment:
I'm not sure if we should note the bug-fix versions for Cassandra and MySQL. Should they be `3.11` and `8.0` respectvively?

Reply:
Cassandra and MySQL compatibility updated with stable versions
MySQL
```
$ brew info mysql@8.0
mysql: stable 8.0.25 (bottled)

$ brew info mysql@5.7
mysql@5.7: stable 5.7.34 (bottled) 
```
<img width="1064" alt="Screen Shot 2021-07-27 at 12 41 00 PM" src="https://user-images.githubusercontent.com/58680313/127091766-7db77a95-cf02-4ec0-a240-bd6a76305b23.png">

Reply:
Thank you for your review comments